viadot/flows/genesys_to_adls.py

```diff
@@ -136,6 +138,8 @@ def __init__(
                 from credentials.
             report_url (str, optional): The url of report generated in json response. Defaults to None.
             report_columns (List[str], optional): List of exisiting column in report. Defaults to None.
+            key_list (List[str], optional): List of keys needed to specify the columns in the GET request method. Defaults to None.
+            conversationId_list (List[str], optional): List of conversationId passed as attribute of GET method. Defaults to None.
```

Comment:
Please change the order of (`conversationId_list`, `key_list`) here or in init parameters to have the same in both places.